modules/installation-network-user-infra.adoc

```diff
@@ -64,6 +64,12 @@ ifeval::["{context}" == "installing-restricted-networks-gcp"]
 :gcp:
 :restricted:
 endif::[]
+ifeval::["{context}" == "installing-rhv-user-infra"]
+:rhv:
+endif::[]
+ifeval::["{context}" == "installing-rhv-default"]
```

Comment:
Should this be for the RHV restricted network assembly instead?
```
ifeval::["{context}" == "installing-rhv-restricted-network"]
````
From what I can tell, this module is not included in the default RHV assembly, so the current ifeval wouldn't make sense.

Reply:
I think you're right. I wish I could remember why I didn't do that in the first place. Fixed.